xml/System.Text.Json/JsonSerializerOptions.xml

```diff
@@ -59,7 +59,7 @@
 
 ## Remarks
 
-By default, `AllowTrailingCommas` is set to `false`, and a <exception cref="T:System.Text.Json.JsonException> is thrown if a trailing comma is encountered.
+By default, `AllowTrailingCommas` is set to `false`, and a <xref:System.Text.Json.JsonException> is thrown if a trailing comma is encountered.
```

Comment:
we should also probably say who throws this exception. This is during deserialization correct?

Reply:
Correct.